NetworkPkg/HttpBootDxe/HttpBootConfig.c

```diff
@@ -634,16 +634,21 @@ HttpBootConfigFormInit (
                       STRING_TOKEN (STR_HTTP_BOOT_CONFIG_FORM_HELP),
                       NULL
                       );
-    UnicodeSPrint (MenuString, 128, L"%s (MAC:%s)", OldMenuString, MacString);
-    HiiSetString (
-      CallbackInfo->RegisteredHandle,
-      STRING_TOKEN (STR_HTTP_BOOT_CONFIG_FORM_HELP),
-      MenuString,
-      NULL
-      );
+    // MU_CHANGE [BEGIN] - CodeQL change
+    if (OldMenuString != NULL) {

```

Comment:
In Ip4ConfigNv.c you have a very similar codepath with also checking MacString for being NULL, should that happen here, too, or can we rely on the StatusNetLibGetMacString to report whether MacString is NULL or not?

Reply:
I think the other instances where I check Macstring should be removed because Macstring will be non-null if we enter this code block. I removed the checks in the other files.